app/CommandHandling/ContextFactory.php

```diff
@@ -10,7 +10,6 @@
 use CultuurNet\UDB3\Http\Auth\Jwt\JsonWebToken;
 use CultuurNet\UDB3\User\CurrentUser;
 use Silex\Application;
-use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\RequestStack;
```

Comment:
I cannot remove this usage until we completely switch to the new router. So I will do that when picking up III-4948 (also added it to the ticket description to be sure)